editor/utils/shortcuts.ts

```diff
@@ -9,9 +10,12 @@ import { useCallback, useLayoutEffect, useRef, useEffect } from "react";
  */
 export const useKeyboardShortcut = (
   // Array of keys that should trigger the callback when used alongside the shift key
-  shortcutKeys: string[],
+  shortcutKeyLookup: ShortcutKeyLookup[],
   // Callback that handles what happens when the keys are pressed
```

Comment:
it would be good to move these comments up into your JSDoc string, or declare an interface with jsdoc comments.